pinot-spi/pom.xml

```diff
@@ -171,6 +171,10 @@
                       <pattern>com.google.common.base</pattern>
                       <shadedPattern>shaded.com.google.common.base</shadedPattern>
                     </relocation>
+                    <relocation>
```

Comment:
do we need to shade both?

Reply:
pinot-avro-base internally depends on pinot-spi. If we don't shade both, pinot-common would still be pulled. Thus, have to shade two modules.